presidio-analyzer/analyzer/__init__.py

```diff
@@ -1,6 +1,7 @@
 import os
 import sys
 
+# pylint: disable=unused-import,line-too-long,wrong-import-position
```

Comment:
ths unused import cant be fixed as we cant remove these lines from the init file.
the wrong position is also a must have because of the 'sys.path.append' line which we must have....